openmdao/solvers/backtracking.py

```diff
@@ -100,5 +128,8 @@ def solve(self, params, unknowns, resids, system, solver, alpha, fnorm,
 
             fnorm = resids.norm()
             if self.options['iprint'] > 0:
-                self.print_norm('BK_TKG', system.pathname, itercount, fnorm,
-                                fnorm0, indent=1, solver='LS')
+                self.print_norm(self.print_name, system.pathname, itercount,
```

Comment:
suggestion: you could remove the first argument (`print_name`) in its many occurrences and have it just once in the `print_norm` method since it's now looks like an attribute thats set for all instances of the base class